src/components/swap/routing/TradeDetails/UbeswapTradeDetails.tsx

```diff
@@ -53,7 +53,7 @@ export const UbeswapTradeDetails: React.FC<Props> = ({ trade, allowedSlippage }:
           <TYPE.black fontSize={14} fontWeight={400} color={theme.text2}>
             Liquidity Provider Fee
           </TYPE.black>
-          <QuestionHelper text="A portion of each trade (0.30%) goes to liquidity providers as a protocol incentive." />
+          <QuestionHelper text="A portion of each trade (0.25%) goes to liquidity providers as a protocol incentive. Another portion of each trade (0.05%) buys back UBE and sends it to the community reserve." />
```

Comment:
The 0.5% fees are not being used to buy back UBE at this point in time. It just gets redirected to the Ubeswap treasury where the community can decide what to do with the rewards.